assets/kube-apiserver/kube-apiserver-deployment.yaml

```diff
@@ -158,6 +158,7 @@ spec:
         - "--openshift-config=/etc/kubernetes/apiserver-config/config.yaml"
 {{- if .KPInfo }}
         - "--encryption-provider-config=/etc/kubernetes/kms-config/config.yaml"
+        - "--encryption-provider-config-automatic-reload=false"
```

Comment:
You need to run `make bindata` in order to regenerate the bindata based on this change.

Reply:
Updated in https://github.com/openshift/ibm-roks-toolkit/pull/947/commits/2dc7e5fb667f5fea8c4cd8f260db9b8d8e473939